packages/terminal/src/browser/terminal-frontend-contribution.ts

```diff
@@ -83,9 +84,14 @@ export class TerminalFrontendContribution implements CommandContribution, MenuCo
         /* Register ctrl + k (the passed Key) as a passthrough command in the
            context of the terminal.  */
         const regCtrl = (k: Key) => {
+            let modifier = isOSX ? Modifier.M4 : Modifier.M1;
+            if (Key.KEY_A.keyCode === k.keyCode) {
```

Comment:
With this, ctrl-A will not be registered as a passthrough keybinding on Mac.  isn't it wrong?  Don't you want to register cmd-A in addition to ctrl-A?

In the xterm.js code, ctrl-A is handled with the rest (it sends a 0x01 byte), and cmd-A is handled in a special way for Mac:
https://github.com/xtermjs/xterm.js/blob/v3/src/Terminal.ts#L1684